backend/Rhyolite/Backend/Logging.hs

```diff
@@ -110,7 +114,9 @@ data LoggingContext m = LoggingContext
 data RhyoliteLogAppender
    = RhyoliteLogAppender_Stderr
    | RhyoliteLogAppender_File FilePath
+#ifdef linux_HOST_OS
    | RhyoliteLogAppender_Journald T.Text -- journalctl log with syslogIdentifier specified.
+#endif
```

Comment:
Instead of removing the constructor would it be better to leave it here and throw an error if it gets used for logging on an unsupported system? As is, this means CPP will bleed to all places that need this.